pkgs/development/python-modules/web3/default.nix

```diff
@@ -16,11 +16,12 @@
   protobuf,
   requests,
   websockets,
+  pyunormalize,
 }:
 
 buildPythonPackage rec {
   pname = "web3";
-  version = "6.5.0";
+  version = "6.20.3";
   format = "setuptools";
```

Comment:
Can we also convert this to pyproject?

Reply:
By doing so, these errors popped up:

```
       > Checking runtime dependencies for web3-6.20.3-py3-none-any.whl
       >   - ckzg not installed
       >   - hexbytes<0.4.0,>=0.1.0 not satisfied by version 1.2.0
       >   - lru-dict not installed
```

Reply:
And as we speak web3 7 doesn't require ckzg anymore.... :/

Reply:
Shall we call it for a day, and fix web3 pyproject project in a future PR? I am happy to do it, and I will keep this in mind.